packages/components/src/lib/controller/overlay/useOverlayController.ts

```diff
@@ -5,15 +5,26 @@ import type { FlowComponentName } from "@/components/propTypes";
 interface Options {
   reuseControllerFromContext?: boolean;
   defaultOpen?: boolean;
+  onOpen?: () => void;
+  onClose?: () => void;
 }
 
```

Comment:
```suggestion
interface Options extends OverlayControllerOptions {
  reuseControllerFromContext?: boolean;
}
```suggestion
interface Options {
  reuseControllerFromContext?: boolean;
  defaultOpen?: boolean;
  onOpen?: () => void;
  onClose?: () => void;
}

```
```